CHANGELOG.md

```diff
@@ -23,6 +23,51 @@ This change was made in [pull request #3599: Enable new link styles by default](
 
 ### Breaking changes
 
+You must make the following changes when you migrate to this release, or your service might break.
+
+#### Update package file paths
+
+To support package dependencies, we’ve moved our source files from `src` into `package/src`. In preparation for additional build targets, our package is now compiled into `package/dist`.
```

Comment:
Is 'our package' the same as `package/src`? And if possible we should phrase the second sentence as an action.

Reply:
Thanks @claireashworth

I'm glad you pointed this out as it's changed more now and needs another tweak!

These are the main two things we've moved, with the old location using strikethrough:

1. ### ~`src`~ → `packages/govuk-frontend/src`
   This is the raw source code for `govuk-frontend`
   It is not included in the package contents [published to npm](https://www.npmjs.com/package/govuk-frontend)

2. ### ~`package`~ → `packages/govuk-frontend/dist`
   This is the built code for `govuk-frontend`
   It is included in the package contents [published to npm](https://www.npmjs.com/package/govuk-frontend)

Where we previously had "our package" as the files in `package`
But soon for v5  "our package" will be `packages/govuk-frontend`

Maybe we don't need to mention the non-action things?

But just tweaking what I have currently would be:

```suggestion
To support package dependencies, we’ve moved our source files from `src` into `packages/govuk-frontend/src`. In preparation for additional build targets, our package is now compiled into `packages/govuk-frontend/dist`.
```

Reply:
Is it important for someone updating file paths to have the info from the first sentence? The only reason to remove it is if it would cause confusion about the action the reader needed to take. 
Would it still be accurate for the second sentence say something like - 'When you're updating your package file path, you can find our package compiled into...'

Reply:
For our contributors yes, but our typical users no.

Let's just keep the second bit where we talk about `dist` since I'd taken influence from this line further down for the v3.0.0 update when `govuk` was similarly added:

https://github.com/alphagov/govuk-frontend/blob/04e5168b534140c5a751f7ded5141148d4d6ef7e/CHANGELOG.md#L1735